README.md

```diff
@@ -53,6 +53,40 @@ Unit tests can ran using the following project level command(s):
 ### Integration Tests
 TBD
 
+## Bootstrap Config
+A bootstrap config file is used to define the initial state of the pessimism service. The file must be `json` formatted with it's directive defined in the `BOOTSTRAP_PATH` env var. 
+
+### Example
+```
+[
+    {
+          "network": "layer1",
+          "pipeline_type": "live",
+          "type": "contract_event", 
+          "start_height": null,
+          "alert_destination": "slack",
```

Comment:
Should alert_destination be an array of strings? What if I want multiple alert destinations for the same invariant?

Reply:
That's a great call-out! I feel like that change falls out-of-scope of this PR though. I can cut an issue in the meantime detailing the change.

Reply:
That's a great call-out! I feel like that change falls out-of-scope of this PR though. I can cut an issue in the meantime detailing the change.

Reply:
#81